_docs/installation.md

```diff
@@ -69,7 +69,7 @@ Add the following lines to your server's configuration block:
     location /api { try_files $uri $uri/ /api.php?$query_string; }
     location /admin { try_files $uri $uri/ /admin.php?$query_string; }
 
-    location /flarum {
+    location ~* ^/(flarum|storage|vendor) {
```

Comment:
@luceos what about the `flarum` executable?

Reply:
gosh how did I miss that 😕

Reply:
Yes I guess it's safer to prevent any execution of that file ? I thought the `htaccess` also blocked it but apparently not...

And I just realized my fix is incomplete, the `composer.json` and `lock` are still accessible... Gonna open another PR